src/scripts/Providers/OSUI/Dropdown/VirtualSelect/scss/_virtualselect.scss

```diff
@@ -379,10 +379,16 @@
 
 // Wrapper options container
 .vscomp-dropbox {
+	--border-color: var(--color-neutral-5);
+	--border-size: 1px;
+
 	background-color: var(--color-neutral-0);
 	border-radius: var(--border-radius-soft);
-	border: var(--border-size-s) solid var(--color-neutral-5);
+	/* use inner shadow to workaround the border issue that causes content to be blurry */
+	box-shadow: inset var(--border-color) 0px 0px 0px var(--border-size);
```

Comment:
can't we remove the `px`? improve the performance a tiny bit

Reply:
Done!